doc/development/tutorials/examples/todo.py

```diff
@@ -43,12 +41,14 @@ def run(self):
         if not hasattr(self.env, 'todo_all_todos'):
             self.env.todo_all_todos = []
 
-        self.env.todo_all_todos.append({
-            'docname': self.env.docname,
-            'lineno': self.lineno,
-            'todo': todo_node.deepcopy(),
-            'target': targetnode,
-        })
+        self.env.todo_all_todos.append(
```

Comment:
This one specific way to format will affect a lot of tests I think, though I'm not sure why this one would have been reformatted. Could we not reformat those kind of expressions because it will probably lead to an explosion of changes.

Reply:
Nope afraid not, there is no way to change this.
I personally prefer this anyway, but it is what it is

Reply:
you could enable the `--preview` flag, which will give you the slightly more condensed version you're after

Reply:
add `preview = true` thanks @danieleades  and it removed the change,
apparently ruff have already addressed your concern @picnixz 😄